scala/finch/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.4.4
```

Comment:
do we really need this file @whiplash ?

Reply:
It can be 1.4.6 (see https://github.com/the-benchmarker/web-frameworks/blob/master/scala/Dockerfile#L1)

Reply:
can not it be removed ? I mean the docker image contains already an `sbt` version